src/assets/scss/templates/_guidelines.scss

```diff
@@ -198,7 +198,7 @@ body {
     position: relative;
     margin: variables.$dp-spaces--lv0;
     padding: variables.$dp-spaces--lv0;
-    color: colors.$dp-color-darkyellow;
+    color: #9758ab;
```

Comment:
You are not using variables for colors anymore?

Reply:
Si, seguimos usando variables, se me paso. Ahora la agrego